src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -214,6 +218,37 @@ class StreamGroup extends React.Component<Props, State> {
     };
   }
 
+  renderReprocessingColumns() {
+    const {data} = this.state;
+    const {statusDetails, count} = data;
+    const {info} = statusDetails;
+
+    const totalEvents = info?.totalEvents ?? 0;
+    const pendingEvents = statusDetails.pendingEvents ?? 0;
+
+    const remainingEventsToReprocess = totalEvents - pendingEvents;
+    const remainingEventsToReprocessPercent =
+      (remainingEventsToReprocess / totalEvents) * 100;
```

Comment:
We have utility for this:
https://github.com/getsentry/sentry/blob/b8bf85948d4c4832c60ab31e9f1a5a3c39fd53cf/src%2Fsentry%2Fstatic%2Fsentry%2Fapp%2Futils.tsx#L139-L146

Reply:
oh nice